### ISSUE TITLE
[turborepo] cli doesn't find package.json in root or any workspaces on windows 10

### ISSUE THREAD

**Issue by digitalmoran** [vercel/turbo#3462]
### What version of Turborepo are you using?

1.7.0

### What package manager are you using / does the bug impact?

pnpm

### What operating system are you using?

Windows

### Describe the Bug

Turbo cli does not work at all on Windows 10.0.19044.2364 Standard User account. Whether using pnpm, npm, or yarn, the turbo cli doesn't find the package.json. Let's say the code is in C:\Projects\monorepo. Every single time it spawns a process it has the wrong process.cwd(). If you put debug code in node_modules\turbo\bin\turbo it outputs C:\Projects as the process.cwd(). Setting the --cwd parameter does not correct the issue. I also tried hard-coding the correct cwd in the turbo js file which gets further, but then winds up in an endless loop because instead of using each package's package.json commands, it uses the top level package.json commands because of the hardcoded cwd. So the tool is completely useless as no commands work. Node version is 16.17.1. I am in an enterprise setting and all of our devs get these exact results.

### Expected Behavior

The turbo cli runs any commands at all as expected because it properly sees and understands the workspace package.json files

### To Reproduce

on Windows 10.0.19044.2364 (I don't know if it's specific to this version)
npx create-turbo@latest and choose pnpm, npm, or yarn

**Comment by NicholasLYang** [comment]
Hi, thanks for the issue! I have a few follow up questions. Do previous versions of turbo work? What is the exact error message that you get? And if you try running turbo with `--skip-infer` at the root of the repository, does that work?

**Comment by digitalmoran** [comment]
I figured out the issue. I installed the exact same configuration in a vanilla Windows 10 VM and then made a change at a time to the OS setup until it matched. I had forgotten that as a part of our development setup we added the following registry values:

`Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Microsoft\Command Processor]
"CompletionChar"=dword:00000009
"PathCompletionChar"=dword:00000009
"DefaultColor"=dword:00000000
"EnableExtensions"=dword:00000001
"AutoRun"="%USERPROFILE%\\dev\\aliases.cmd"`

That runs aliases.cmd every time a new command shell is opened for the purpose of adding custom aliases. Inside of that file someone had placed a cd command to have each shell open to %USERPROFILE%\Projects.  Removing that line resolves the issue. I know this is an edge case but maybe a tiny note in the documentation about it is warranted since it totally breaks the cli in both cmd and git bash. I understand that it might break other things too, but in our case nothing else had been encountered in over 4 years that made us catch this issue.
